hl7/4.2/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,174 @@
+= HL7 EDI Connector 4.2 - Mule 4
+:page-aliases: connectors::hl7/hl7-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Anypoint Connector HL7 EDI (HL7 EDI Connector) is a collection of features that facilitate integration with healthcare systems by providing the tools needed to easily create, read and transform HL7 version v2.x messages within Mule. Health Level Seven International (HL7) is a standards development organization that defines a set of standards for the transport and communication of healthcare information.
+
+For compatibility information and fixed issues, see the HL7 EDI Connector Release Notes.
+
+Standard schema definitions are included in HL7 EDI Connector for
+HL7 versions 2.1, 2.2, 2.3.1, 2.3, 2.4, 2.5, 2.5.1, 2.6, 2.7,
+2.7.1, 2.8, 2.8.1. The schemas are provided in two forms:
+
+* Standard HL7: using required or optional status data types,
+field lengths, and repetition counts (as listed) for each HL7 version. +
+
+* Lax HL7: all segments and value items are optional. These
+low-level data types are treated as simple strings. Field lengths and repetition counts are not enforced.
+
+Both forms of schemas are available in the connector `classpath`. Standard HL7
```

Comment:
...in the connector `classpath` <what? What is the object of the syntax `classpath`?>